src/Server.php

```diff
@@ -1192,4 +1192,22 @@ public function handlePhpErrors($errno, $errstr)
     {
         throw $this->fault($errstr, 'Receiver');
     }
+
+    /**
+     * Disable the ability to load external XML entities based on libxml version
+     *
+     * If we are using libxml < 2.9, unsafe XML entity loading must be
+     * disabled with a flag.
+     *
+     * If we are using libxml >= 2.9, XML entity loading is disabled by default.
+     *
+     * @return bool
+     */
+    public static function disableEntityLoader($flag = true)
```

Comment:
Since it is only needed for internal use of this class, I propose the following:
```suggestion
    private function disableEntityLoader($flag = true)
```